pegasus/sites.v3/code.org/public/js/volunteer-local.js

```diff
@@ -32,6 +35,21 @@ $(function() {
     .change(function() {
       initializeMap();
     });
+
+  if (window.location.search.indexOf("mapbox") !== -1) {
+    mapboxGeocoder = new MapboxGeocoder({
+      accessToken: mapboxgl.accessToken,
+      types: "country,region,place,postcode,locality,neighborhood"
+    });
+
+    mapboxGeocoder.addTo("#mapbox-geocoder");
+    mapboxGeocoder.on("result", function(result) {
+      const coordinates = result.result.geometry.coordinates;
+      gmap_loc = coordinates[1] + "," + coordinates[0];
```

Comment:
Oh how I love that Google and Mapbox need their coordinates in different orders 🙃